Automattic/wp-calypso
# Clean up product names translations

Related to https://github.com/Automattic/jetpack-marketing/issues/581

## Proposed Changes

* Remove translations that are not used or are redundant
* Do not translate product short names

## Testing Instructions

* Go to http://jetpack.cloud.localhost:3001/es/pricing and make sure no product names are translated

## Pre-merge Checklist
